napari/utils/shortcuts.py

```diff
@@ -12,4 +12,32 @@
     'napari:transpose_axes': ['Control-T'],
     'napari:toggle_grid': ['Control-G'],
     'napari:toggle_selected_visibility': ['V'],
+    'napari:activate_label_erase_mode': ['E'],
+    'napari:activate_fill_mode': ['F'],
+    'napari:activate_paint_mode': ['P'],
+    'napari:activate_label_pan_zoom_mode': ['Z'],
+    'napari:activate_label_picker_mode': ['L'],
+    'napari:new_label': ['M'],
+    'napari:decrease_label_id': ['D'],
+    'napari:increase_label_id': ['I'],
+    'napari:activate_points_add_mode': ['P'],
+    'napari:activate_points_select_mode': ['S'],
+    'napari:activate_points_pan_zoom_mode': ['Z'],
+    'napari:select_all': ['A'],
+    'napari:delete_selected_points': ['Backspace', 'Delete'],
+    'napari:activate_add_rectangle_mode': ['R'],
+    'napari:activate_add_ellipse_mode': ['E'],
+    'napari:activate_add_line_mode': ['L'],
+    'napari:activate_add_path_mode': ['T'],
+    'napari:activate_add_polygon_mode': ['P'],
+    'napari:activate_direct_mode': ['D'],
+    'napari:activate_select_mode': ['S'],
+    'napari:activate_shape_pan_zoom_mode': ['Z'],
+    'napari:activate_vertex_insert_mode': ['I'],
+    'napari:activate_vertex_remove_mode': ['X'],
+    'napari:copy_selected_shapes': ['Control-C'],
+    'napari:paste_shape': ['Control-V'],
+    'napari:select_all_shapes': ['A'],
+    'napari:delete_selected_shapes': ["Backspace", "Delete"],
+    'napari:finish_drawing_shape': ['Escape'],
```

Comment:
+1, easy and strait forward and good.
I guess that was a lot of not so easy copy and past.

For another PR do we want to make the values tuples to avoid unwanted mutations ? Not sure we can make the dict immutable.

Reply:
>For another PR do we want to make the values tuples to avoid unwanted mutations ? Not sure we can make the dict immutable.

Good call!

Reply:
> I guess that was a lot of not so easy copy and past.
It was a lot of copy paste!  😆 

And yes, I agree about the tuples.  :)